### ISSUE TITLE
Backticks being forbidden

### ISSUE THREAD

**Issue by ErikEvenson** [clutchski/coffeelint#313]
Coffeelint is complaining that `Backticks are forbidden` for this snippet (which is perfectly legal):

```
    getUuid: ->
      `'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
          var r = Math.random()*16|0, v = c == 'x' ? r : (r&0x3|0x8);
          return v.toString(16);
      });`
```

**Comment by ImaginaryDevelopment** [comment]
add it where? please include context when answering questions/issues.
